pkg/cmd/pipelinerun/log_writer.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelinerun
+
+import (
+	"fmt"
+
+	"github.com/tektoncd/cli/pkg/cli"
+)
+
+type LogWriter struct{}
+
+func (lw *LogWriter) Write(s *cli.Stream, logC <-chan Log, errC <-chan error) {
+	for logC != nil || errC != nil {
+		select {
+		case l, ok := <-logC:
+			if !ok {
+				logC = nil
+				continue
+			}
+
+			if l.Log == "LOGEOF" {
```

Comment:
This isn't true anymore .. Do we have a special "LOGEOF" ? If so, perhaps we should think of passing "events" in a better way .. events: {started, ended..}

Reply:
> `This isn't true anymore .` 

:thinking: